src/main_operations/modules/apriltags/pre_processing/ai_acceleration/training/augmentation/visualize_data.py

```
@@ -0,0 +1,89 @@
+    json_path = image_path.replace(".png", ".json")
+
+    # Check if JSON file exists
+    if not os.path.exists(json_path):
+        print(f"Error: JSON file not found at {json_path}")
+        return
+
+    # Load and parse JSON data
+    try:
+        with open(json_path, "r") as file:
+            detection_data = json.load(file)
+    except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)